examples/dataflow-data-generator/data-generator-pipeline/tests/test_get_pyarrow_translated_schema.py

```diff
@@ -0,0 +1,77 @@
+import json
```

Comment:
change request in original PR: https://github.com/GoogleCloudPlatform/professional-services/pull/199#discussion_r262759143